### PR TITLE
Implement stack based approach.

### PR DESCRIPTION
**Pull request by qnikst**

### CODE REVIEW

File: cloud-haskell.cabal [hunk 1]
```diff
@@ -47,10 +47,10 @@ Flag p2p
   Manual:  True
 
 Library
-  Build-depends:     
+  Build-depends:
                      rank1dynamic >= 0.2.0.0 && < 0.3,
                      distributed-static >= 0.3.0.0 && < 0.4,
-                     network-transport >= 0.4.1.0 && < 0.5,
+                     network-transport >= 0.4.1.0 && < 0.5
```
Comment (mboes) [inline cloud-haskell.cabal]: Have you really checked that this still compiles? It seems that the only change on this line introduces a syntax error.

File: stack.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    commit: master
+- location:
+    git: git@github.com:haskell-distributed/distributed-process
+    commit: master
+#- location:
+#    git: git@github.com:haskell-distributed/distributed-process-platform
+#    commit: master
+- location:
+    #git: git@github.com:haskell-distributed/distributed-process-extras
+    git: git@github.com:qnikst/distributed-process-extras
```
Comment (mboes) [inline stack.yaml:33]: why the location change? Let's get this merged once upstream changes are merged.

File: stack.yaml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+#- location:
+#    git: git@github.com:haskell-distributed/distributed-process-azure
+#    commit: master
+#- location:
+#    git: git@github.com:haskell-distributed/azure-service-api
+#    commit: master
+#- location:
+#    git: git@github.com:haskell-distributed/distributed-process-demos
+#    commit: master
+
```
Comment (mboes) [inline stack.yaml]: extraneous whitespace

File: stack.yaml (hunk 2 above)
Comment (qnikst) [inline stack.yaml:33]: Agree, the main purpose of this PR was to give an ability to easily test all the changes .

Comment (qnikst) [inline stack.yaml:33]: Another approach is to send branch to upstream repo, as I have access to that repo now.

Comment (mboes) [inline stack.yaml:33]: yes.

**Comment by facundominguez** [comment]
Doesn't find `rematch` a dependency of`d-p-test` on the nightly stackage (because it is not there).

**Comment by mboes** [comment]
a511cf27269770bdd3ec75ce307f1aea0a7803da introduced a `stack.yaml` file in master.

**Pull request closed without merging**